### ISSUE TITLE
Create example with non-API restricted imagery

### ISSUE THREAD

**Issue by drewbo** [developmentseed/label-maker#17]
Create an example using non-API restricted imagery so users can start using the tool more easily:
- [South Africa Aerial](http://aerial.openstreetmap.org.za/)
- [OpenAerialMap](https://openaerialmap.org/)

**Comment by Geoyi** [comment]
Good idea, @drewbo, it could worth to look into [DG's Open Data Program](https://www.digitalglobe.com/opendata/) too.
